Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+## Coaching Guide
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
```

Comment:
they are on a project now and I think they have specs as of this week? @jrob8577 @shereefb 

The coaching goal currently links to [this other guide](https://dianavashti.gitbooks.io/how-to-get-the-most-out-of-coaching/content/) should we merge them?